apps/src/templates/studioHomepages/teacherHomepageV2/SectionOptionsDropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import {ActionDropdown} from '@code-dot-org/component-library/dropdown';
+import {ActionDropdownOption} from '@code-dot-org/component-library/dropdown/actionDropdown';
+import React from 'react';
+import {useNavigate} from 'react-router-dom';
+
+import {
+  TEACHER_NAVIGATION_SECTIONS_URL,
+  TEACHER_NAVIGATION_PATHS,
+} from '@cdo/apps/templates/teacherNavigation/TeacherNavigationPaths';
+import i18n from '@cdo/locale';
+
+import styles from './teacherHomepage.module.scss';
+
+export interface SectionOptionsDropdownProps {
+  sectionId: number;
+}
+
+export const SectionOptionsDropdown: React.FC<SectionOptionsDropdownProps> = ({
+  sectionId,
+}) => {
+  const navigate = useNavigate();
+
+  const onSectionSettingsClick = () => {
```

Comment:
Can these all be static functions that take the section ID? That will save some memory and updates.